examples/webgl_effects_displacement.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - effects - displacement</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				background: #000;
+				color: #fff;
+				padding: 0;
+				margin: 0;
+				overflow: hidden;
+				font-family: georgia;
+				text-align: center;
+			}
+
+			#info {
+				position: absolute;
+				width: 100%;
+				top: 10px;
+			}
+
+			a {
+				color: skyblue;
+				text-decoration: none
+			}
+		</style>
+	</head>
+
+	<body>
+
+		<div id="info">
+			Depth Displacement Effect<br/> Code & texture by <a href="http://orfleisher.com">juniorxsound</a>
+		</div>
+
+		<script src="../build/three.js"></script>
+
+		<script src="js/effects/DepthDisplacement.js"></script>
+
+		<script src="js/Detector.js"></script>
+
+		<script>
+			if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+			var container, stats;
+
+			var camera, scene, renderer, effect;
+
+			init();
+
+			function init() {
+
+				container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 1, 100 );
+				camera.position.set( 0, 0, 1 );
+
+				scene = new THREE.Scene();
+
+				renderer = new THREE.WebGLRenderer();
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				container.appendChild( renderer.domElement );
+
+				var colorTexture, depthTexture;
+
+				var loadingManager = new THREE.LoadingManager();
+				var textureLoader = new THREE.TextureLoader( loadingManager );
+
+				textureLoader.load( './textures/depth_effect_depthmap.png', function ( texture ) {
+
+					depthTexture = texture;
+					depthTexture.minFilter = THREE.NearestFilter;
+					depthTexture.magFilter = THREE.LinearFilter;
+					depthTexture.format = THREE.RGBFormat;
+					depthTexture.generateMipmaps = false;
+
+				} );
+
+				textureLoader.load( './textures/depth_effect_color.png', function ( texture ) {
+
+					colorTexture = texture;
+					colorTexture.minFilter = THREE.NearestFilter;
+					colorTexture.magFilter = THREE.LinearFilter;
+					colorTexture.format = THREE.RGBFormat;
+					colorTexture.generateMipmaps = false;
+
+				} );
+
+				loadingManager.onLoad = function () {
+
+					var rendererSize = renderer.getSize();
+
+					effect = new THREE.DepthDisplacementEffect( renderer, colorTexture, depthTexture );
+					effect.setSize( rendererSize.width, rendererSize.height, window.innerWidth, window.innerHeight );
+					scene.add( effect.getObject3D() );
```

Comment:
@Mugen87 This PR has an unusual API, but I will leave that up to you.

Reply:
@WestLangley I am open to suggestions/examples for design patterns for such an effect, given it is applied on a `PlaneGeometry` I picked the obvious of just adding it to the scene, but perhaps there is a simpler way of applying it?

Reply:
If possible, remove `updateMousePosition()`, `setDirection() and `setDisplacement()` from `DepthDisplacementEffect`.

Reply:
It is an interesting effect, but maybe the reason it doesn't seem to fit in is because it does not require three.js. Maybe it is more appropriate as a [ShaderToy](https://www.shadertoy.com) example.

I guess you could render a three.js scene, and use the scene's depth map, instead... I'm not sure what the point of that would be, though...

Reply:
@WestLangley Thanks! I will look into sharing this on ShaderToy

@Mugen87 Thanks for the suggestions! pushed some changes -https://github.com/mrdoob/three.js/pull/14479/commits/cf4d487e4d7fa28ea2dbe97c056ec36928001cca